pkg/utils/utils.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021 Nordix Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Go version 1.10 or greater is required. Before that, switching namespaces in
+// long running processes in go did not work in a reliable way.
+// +build go1.10
+
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var (
+	// DefaultCNIDir used for caching hostIFName
+	DefaultCNIDir = "/var/lib/cni/ovs"
+)
+
+// SaveConf takes in container ID, data dir and Pod interface name as string and a json encoded struct Conf
```

Comment:
This says it takes data dir. However, it is not in the function signature. Also it does not document what does the `prefix` stand for

Reply:
updated method comment, hope its relevant now :)